src/ray/pubsub/publisher.cc

```diff
@@ -21,13 +21,16 @@ namespace pubsub {
 namespace pub_internal {
 
 template <typename KeyIdType>
-void SubscriptionIndex<KeyIdType>::AddEntry(const std::string &key_id_binary,
+bool SubscriptionIndex<KeyIdType>::AddEntry(const std::string &key_id_binary,
```

Comment:
If it's not expected to return false, should we add a debug log for this?

Reply:
It is okay to return false. Since it seems like there are times the registerSubscription is called multiple times. false simply means the function is no-op